yarn-project/noir-contracts/contracts/docs_example_contract/src/types/transparent_note.nr

```diff
@@ -0,0 +1,139 @@
+// docs:start:token_types_all
+use dep::aztec::{
+    note::{
+        note_header::NoteHeader,
+        note_interface::NoteInterface,
+        utils::compute_siloed_note_hash,
+    },
+    hash::{compute_secret_hash, pedersen_hash},
+    context::PrivateContext,
+};
+
+global TRANSPARENT_NOTE_LEN: Field = 2;
+
+// Transparent note represents a note that is created in the clear (public execution),
+// but can only be spent by those that know the preimage of the "secret_hash"
+struct TransparentNote {
```

Comment:
This is now copied in multiple places so I would move this to aztec-nr and import it from there.

Reply:
On this. We need some more consistency on what belongs in aztec-nr and what don't. @rahul-kothari do you have a clear idea or partitioning? Feels wrong with aztec-nr that is so huge that it is impossible to figure out and have all kinds of things that might not really be needed

Reply:
we can always add it like with value-note -> technically not in `aztec-nr/src/aztec` but still in a different workspace